src/routes/components/Checkbox.tsx

```diff
@@ -152,6 +160,7 @@ export default function CheckboxPage() {
             </Sandbox>
 
             <ComponentProperties properties={componentProperties} />
+            <CheckboxExamples />
```

Comment:
I put the example in its own file to keep the main doc shorter. Is this preferred? Or do we only split it out if the examples get too long?